packages/trader/src/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -319,6 +325,8 @@ const ReplayChart = connect(({ modules, ui, common }) => {
         barriers_array: contract_store.barriers_array,
         markers_array: contract_store.markers_array,
         symbol: contract_store.contract_info.underlying,
+        contract_info: contract_store.contract_info,
+        all_ticks: contract_store.contract_info.audit_details.all_ticks,
```

Comment:
we have to use key **all_ticks** instead of **tick_stream** because for draw chart required 2 ticks before the first tick and 1 tick after the last tick. for example, a contract with 5 ticks requires 8 ticks to draw a chart, they are in object **all_ticks**